src/Pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+const Homepage = () => {
+  return (
+    <div>
+      <img
+        className="cover"
+        src={require("../../Assets/cover.jpg")}
+        alt="homepage"
+      />
+      <div className="cover-wrapper">
+        <div className="cover-text">FOR COLOURFUL STEPS IN YOUR LIFE</div>
+        <div className="btn-wrapper">
+            <button className="cover-btn">Explore</button>
+        </div>
+        
+      </div>
+
+      <div className="secondary-cover-container">
+        <div>
+          <img
+            className="secondary-cover"
+            src={require("../../Assets/secondary-cover-1.jpg")}
```

Comment:
You can use a media manager like cloudinary to host all the images and link the in the src